css/style.css

```diff
@@ -264,7 +264,62 @@ hr {
     margin-top: 50px;
 }
 
+/******************Section-Sharpen your skills*****************/
+.section table{
+  border-collapse: collapse;
+  border-spacing: 0;
+  width: 90%;
+  border: 1px solid #ddd;
+  align: centre;
+}
+
+center {
```

Comment:
`center` tag by default provides centering so no need to give margin left and margin right auto.